lib/datura/to_es/vra_to_es/fields.rb

```diff
@@ -116,16 +116,16 @@ def person
       # subject element
       if get_text("@type", xml: p) == "personalName"
         {
-          id: nil,
-          name: get_text(".", xml: p),
-          role: nil
+          "id" => nil,
+          "name" => get_text(".", xml: p),
+          "role" => nil
```

Comment:
Ah, a "symbol" vs "string" as hash key issue. These aren't as common if everything is working solely within Ruby and not interacting with other data formats or systems. Hope this wasn't too big of a nightmare to figure out, but good to understand the distinction to have it in mind for future troubleshooting :bulb: